crates/bitwarden-exporters/src/cxf/export.rs

```diff
@@ -78,15 +78,13 @@ impl From<CipherType> for Vec<Credential> {
     fn from(value: CipherType) -> Self {
         match value {
             CipherType::Login(login) => (*login).into(),
-            // TODO(PM-15450): Add support for credit cards.
             CipherType::Card(card) => (*card).into(),
             // TODO(PM-15451): Add support for identities.
             CipherType::Identity(_) => vec![],
             // Secure Notes only contains a note field which is handled by `TryFrom<Cipher> for
             // Item`.
             CipherType::SecureNote(_) => vec![],
-            // TODO(PM-15448): Add support for SSH Keys.
-            CipherType::SshKey(_) => vec![],
+            CipherType::SshKey(ssh) => (*ssh).try_into().unwrap_or_default(),
```

Comment:
Should we change this From impl to TryFrom to not ignore the error here? This impl is only used inside `impl TryFrom<Cipher> for Item {` so we should be able to handle it there.

Reply:
I think we will eventually want to handle the errors here. If we are unable to import an SSH key that should bubble up to the UI somehow.